makefile

```diff
@@ -1,17 +1,5 @@
 test: heroku-22-build heroku-20-build heroku-18-build
 
-build:
```

Comment:
Could you update these to build the Rust binary instead?

At the moment it seems there is no command or documented steps for building the Rust binary, which means that it will be a lot harder for anyone who isn't you to work out how to do it. 

Even if they are familiar with Rust, they might also forget to pass `--release` etc, so I think it's worth having a Makefile target for this. (It would also allow me as reviewer to tell whether the binary vendored in this PR was built in release mode)

Reply:
@joshwlewis I haven't tried this locally, but I wonder if the `--root` arg of `cargo install` (along with the `--git` arg) could be used to install the binary into this buildpack, without needing to manually git clone?
https://doc.rust-lang.org/cargo/commands/cargo-install.html